vendor/assets/javascripts/schedule/schedule.js

```diff
@@ -51,6 +51,10 @@ window.Schedule = function (scheduleContainer, options) {
     eventElement.style.borderColor        = BORDER_COLORS[colorIndex];
     eventBackground.style.backgroundColor = BACKGROUND_COLORS[colorIndex];
 
+    if("tooltip" in event) {
```

Comment:
Making these changes because all the other Javascript code apparently conforms to these standards, but as stated the standards are informal and not forced or documented.